javascript/javascript1/week2/Homework.js

```diff
@@ -0,0 +1,146 @@
+/* Step 2: Javascript warm up part one
+              --FreeCodeCamp--
+      https://www.freecodecamp.org/BogdyAlex87*/
+
+
+    //********* */ Flight booking fullname function*************
+              https://codepen.io/Bogdy-Alex/pen/zYXQxWa        
+
+    // const fullname1 = "Benjamin"
+    // const fullname2 = "Hughes"
+
+    function getFullname (firstname,surname,gender,useFormalName=false) {
+      if(useFormalName) {
+        if(gender === "male") {
+        return "Lord "+`${firstname} ${surname}`;
+    } else if (gender === "female"){
+      return "Lady " + `${firstname} ${surname}`;
+    } 
+  } else {
+      return `${firstname} ${surname}`;
+    }
+  }; 
+
+    console.log(getFullname("Benjamin", "Hughes","male",true));
+    console.log(getFullname("Oana", "Loredana","female", true));
+    console.log(getFullname(" "," ","female",true));
+
+    // Event application
+
+function getEventWeekday (daysFromToday) {
+  const weekdays = ["Monday","Tuesday" , "Wednesday" , "Thursday" , "Friday" , "Saturday" , "Sunday"];
+  const today = weekdays.indexOf("Sunday");  //I can choose any day from the list
+  const eventDay =  (today + daysFromToday) % 7;
+  return weekdays[eventDay];
+};
+console.log(getEventWeekday(5)); // Logs out "Friday"
+
+
+    //*********** */ Weather wear********
+
+    function howIsWeather(temperature) {
+      if (temperature >= 30 ) {
+         return `The temperature is ${temperature} degrees, we recommand to wear shorts and t-shirt.`;
+    }else if (temperature >= 15 && temperature < 15) {
+         return  `The temperature is ${temperature} degrees, we recommand to wear pants and blouse.`;
+    }else {
+         return `The temperature is ${temperature} degrees ,you should wear jacket and gloves.`;
+    }
+  }
+
+    let getRandomDegrees = Math.floor(Math.random() * (40 - (-10) + 1)) + (-10); //return a random nr between -10 and 40.
+
+    const clothesToWear = howIsWeather(getRandomDegrees)
+    console.log(clothesToWear);
+
+
+                // ********** Student manager*************
+
+  const class07Students = [];
+  function addStudentToClass(studentName) {
```

Comment:
I like what you've done in this function. Very tidy and easy to read. Also good code comments.